packages/core/addon/components/navi-visualizations/goal-gauge.ts

```diff
@@ -9,18 +9,15 @@
  *  }}
  */
 
-import { alias } from '@ember/object/computed';
+import { alias, readOnly } from '@ember/object/computed';
```

Comment:
Still using `alias` in some places so either remove those properties and grab them from `this.config.*` or change them to `readOnly`